tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -1,98 +1,148 @@
+  const [
+    mapZoom,
+    setZoom,
+  ] = useLocalState(context, 'number', 1);
+  let body;
+  // Data view
+  if (tabIndex === 0) {
+    body = (
+      <Table>
+        <Table.Row header>
+          <Table.Cell>
+            Name
+          </Table.Cell>
+          <Table.Cell>
+            Status
+          </Table.Cell>
+          <Table.Cell>
+            Location
+          </Table.Cell>
+        </Table.Row>
+        {crew.map(cm => (
+          <Table.Row key={cm.name}>
+            <TableCell>
+              {cm.name} ({cm.assignment})
+            </TableCell>
+            <TableCell>
+              <Box inline
+                color={cm.dead ? 'red' : 'green'}>
+                {cm.dead ? 'Deceased' : 'Living'}
+              </Box>
+              {cm.sensor_type >= 2 ? (
+                <Box inline>
+                  {'('}
+                  <Box inline
+                    color={COLORS.damageType.oxy}>
+                    {cm.oxy}
+                  </Box>
+                  {'|'}
+                  <Box inline
+                    color={COLORS.damageType.toxin}>
+                    {cm.tox}
+                  </Box>
+                  {'|'}
+                  <Box inline
+                    color={COLORS.damageType.burn}>
+                    {cm.fire}
+                  </Box>
+                  {'|'}
+                  <Box inline
+                    color={COLORS.damageType.brute}>
+                    {cm.brute}
+                  </Box>
+                  {')'}
+                </Box>
+              ) : null}
+            </TableCell>
+            <TableCell>
+              {cm.sensor_type === 3 ? (
+                data.isAI ? (
+                  <Button fluid
+                    icon="location-arrow"
+                    content={
+                      cm.area + " (" + cm.x + ", " + cm.y + ")"
+                    }
+                    onClick={() => act('track', {
+                      track: cm.ref,
+                    })} />
+                ) : (
+                  cm.area + " (" + cm.x + ", " + cm.y + ")"
+                )
+              ) : "Not Available"}
+            </TableCell>
+          </Table.Row>
+        ))}
+      </Table>
+    );
+  // Map view
+  } else if (tabIndex === 1) {
+    body = (
+      (
+        <Box textAlign="center">
+          Zoom Level:
+          <NumberInput
+            animated
+            width="40px"
+            step="0.5"
+            stepPixelSize="10"
+            value={mapZoom}
+            minValue="1"
+            maxValue="8"
+            onDrag={(e, value) => setZoom(value)} />
+          <NanoMap zoom={mapZoom}>
+            {crew.filter(x => x.sensor_type === 3).map(cm => (
+              <NanoMap.Marker
+                key={cm.ref}
+                x={cm.x}
+                y={cm.y}
+                zoom={mapZoom}
+                icon="circle"
+                tooltip={cm.name}
+                color={cm.dead ? 'red' : 'green'}
+              />
```

Comment:
When I tested locally, markers were visible at zoom level 1x then disappeared if I increased it by even 0.1.

Reply:
Cannot reproduce